build.rs

```diff
@@ -6,12 +6,15 @@ extern crate pkg_config;
 #[cfg(unix)]
 #[cfg(feature = "decode_test")]
 extern crate bindgen;
+extern crate nasm_rs;
 
 use std::env;
 use std::fs;
 use std::path::Path;
 
 fn main() {
+    nasm_rs::compile_library_args("librav1easm.a", &["src/x86/mc.asm"], &["-Isrc/"]);
```

Comment:
This should be guarded so only x86* would use it. add another `-I` to find the config.asm or pass its symbols directly.